cli/atlas/templates/pkg/svc/zserver_test.go.gotmpl

```diff
@@ -0,0 +1,33 @@
+package svc
+
+import (
+	"context"
+	"testing"
+	"github.com/golang/protobuf/ptypes/empty"
+)
+
+func TestGetVersion(t *testing.T) {
+	var tests = []struct {
+		expected *pb.VersionResponse
+		err      error
+	}{
+		{
+			expected: &pb.VersionResponse{Version: version},
+			err:      nil,
+		},
+	}
+	for _, test := range tests {
```

Comment:
I love the idea of providing a default test file to show the proper convention for building tests in Go. It turns out we've been doing them wrong, however: https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks

wrap the body of this `for` loop in a `t.Run` call so we do it the way the docs do it.

Reply:
Good find!